lua/plenary/busted.lua

```diff
@@ -148,6 +159,14 @@ mod.inner_describe = function(desc, func)
   end
 end
 
+mod.before_each = function(fn)
+  table.insert(current_before_each[current_description[#current_description]], fn)
+end
+
+mod.clear = function()
+  vim.api.nvim_buf_set_lines(0, 0, -1, false, {})
+end
```

Comment:
What do we need to clear?

Reply:
Later we can do something like:

Go through all the buffers, delete them, etc.